examples/users/hydra.yaml

```diff
@@ -0,0 +1,11 @@
+connection:
+  uri: mongodb://localhost:27017/admin
+users:
+- user: 'david'
+  pwd: 'pass123'
```

Comment:
How about making this `password` instead? `pwd` requires the user to remember this abbreviation. See https://github.com/ryanmcdermott/clean-code-javascript#use-meaningful-and-pronounceable-variable-names

Reply:
It's pwd in mongo. So we have the question here to follow their naming or use better naming. In general I feel we should try use upstream terminologies as much as possible.

But this annoyed me too to be fair I originally had it as password and changed it. 

Let me know your thoughts.